agent/src/agent/pipeline/config/handlers/influx.py

```diff
@@ -156,28 +94,15 @@ def update_source_configs(self):
                     'dimensions': columns,
                     'metric': measurement_name,
                     'delay': delay,
-                    'interval': str(interval) + 's',
+                    # interval is in minutes
```

Comment:
why interval in minutes? don't change any config to not break backward compatibility

Reply:
omg, I thought it was in minutes